pkg/privatelinks/scope.go

```diff
@@ -64,11 +64,10 @@ func (s *Scope) LookupPrivateLink(privateLinkResourceID string) (capz.PrivateLin
 func (s *Scope) GetPrivateLinksWithAllowedSubscription(managementClusterSubscriptionID string) []capz.PrivateLink {
 	var privateLinksWhereMCSubscriptionIsAllowed []capz.PrivateLink
 	for _, privateLink := range s.privateLinks {
-		if slice.Contains(privateLink.AllowedSubscriptions, managementClusterSubscriptionID) {
+		if util.ContainsPtr(privateLink.AllowedSubscriptions, managementClusterSubscriptionID) {
```

Comment:
Same here, slice.Contains in `azure-private-endpoint-operator` but not Contains for Pointer